src/components/DesktopNewsletter/index.jsx

```diff
@@ -0,0 +1,14 @@
+
+
+export default function DesktopNewsletter() {
+    return(
+    <div className="grid h-screen bg-primary place-items-center">
+        <div className="relative h-4/6 w-1/2 bg-primary-dark rounded-[2.813rem] right-20 bottom-8"/>
+        <div className="absolute  h-4/6 w-1/2 pl-7 bg-secondary-light rounded-[2.813rem]"> 
+            <h1 className="mt-14"> Join Our Newsletter!</h1>
+            <p className="w-5/6 "> Want to get updates as fast as possible! Join our newsletter! We will send details about the event with sign up and RSVP forms as we get closer to the start date! We will also send insights into what types of events we are planning, and much more! Sign up to make sure you get the latest as soon as possible! </p>
```

Comment:
Stylistic thing and I know it was not in the Figma design, but could we lighten up the paragraph text a bit to make it less contrasty on the eyes?